src/Features/Core/Portable/SolutionCrawler/ISolutionCrawlerProgressReporter.cs

```diff
@@ -26,19 +26,25 @@ internal interface ISolutionCrawlerProgressReporter
     internal struct ProgressData
     {
         public ProgressStatus Status { get; }
-        public string FilePathOpt { get; }
 
-        public ProgressData(ProgressStatus type, string filePathOpt)
+        /// <summary>
+        /// number of pending work item in the queue
+        /// </summary>
+        public int? PendingItemCount { get; }
+
+        public ProgressData(ProgressStatus type, int? pendingItemCount)
         {
             Status = type;
-            FilePathOpt = filePathOpt;
+            PendingItemCount = pendingItemCount;
         }
     }
 
     internal enum ProgressStatus
     {
         Started,
-        Updated,
+        Paused,
+        PendingItemUpdated,
```

Comment:
📝 It is not obvious to me what _this specific item_ is. The others were clear by their name.

Reply:
so what is your suggestion? PendingItemUpdated means number of pending item count is updated.

Reply:
updated to PendingItemCountUpdated

Reply:
My suggestion was to document it. The rename is sufficient to clarify it for me though.